code/game/machinery/doors/firedoor.dm

```diff
@@ -24,6 +26,8 @@
 	var/nextstate = null
 	var/boltslocked = TRUE
 	var/list/affecting_areas
+	///the number of people trying to open this firelock without any tools
+	var/number_of_people_trying_to_open = 0
```

Comment:
For the sake of logs and better messaging (saying *who*, not simply "others"), this should probably be a lazy list of the people who are trying to open it. LAZYNULL on Destroy(), of course, and for best peace-of-mind WRT hard dels, either making them weak references/using their `REF` (easier) or hooking signals onto their destruction.